ui/desktop/app/routes/scopes.js

```diff
@@ -6,14 +6,35 @@
 import Route from '@ember/routing/route';
 import { A } from '@ember/array';
 import { inject as service } from '@ember/service';
+import { set, get } from '@ember/object';
```

Comment:
Are these `set` and `get` required instead of just accessing like normal?

Reply:
Removed use of `set` and `get`.